Sources/Extensions/UIFont+Components.swift

```diff
@@ -12,7 +12,7 @@ public extension UIFont {
     }
 
     public static var title2: UIFont {
-        let font = UIFont(name: "FINNTypeWebStrippet-Light", size: 28) ?? UIFont.systemFont(ofSize: 28, weight: .light)
+        let font = UIFont(name: "FINNTypeWebStrippet-Medium", size: 24) ?? UIFont.systemFont(ofSize: 24, weight: .medium)
```

Comment:
Is this in line with the FINN-fonts from Troika?

Reply:
I don't think so, need to talk with Ingrid to decrease fonts across Finnivers first.

Reply:
Rolled back to previous values.